### PR TITLE
Koa support

### PR DESCRIPTION
**Pull request by inadarei**

### CODE REVIEW

File: examples/serverKoa.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 
-app.listen(3535);
+app.use(check.koa());
+
+const check2 = healthcheck({"path" : "/ping"});
+app.use(check2.koa());
+
+app.listen(3535, () => console.log('Example app listening on port 3535!'));
+
+//app.listen(3535);
```
Comment (justinbach) [inline examples/serverKoa.js]: Old code?

File: test/acceptance/koa-basic-test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    const res2 = await nf(`${baseuri}/health`);  
+    t.equal(res2.status, 200, 'proper http status code for /health');
+    t.equal(res2.headers.get('content-type'), 
+            'application/health+json; charset=utf-8',
+            'proper content type for health endpoint');
+    const response = await res2.json();
+    t.same(response.status, 'pass',
+      'healthcheck endpoint status works');
+    //t.same(response.details["cassandra:timeout"].metricUnit, 'ms',
+    //  'healthcheck endpoint details work');
```
Comment (justinbach) [inline test/acceptance/koa-basic-test.js]: Should this assertion be fixed or deleted?

File: test/acceptance/koa-basic-test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  // avoid problems if this env var is already set from wherever test was run
+  process.env.NODE_HEALTH_ENDPOINT_PATH = "";
+
+  const server = getServer();
+  const util = require('../support/util');
+  const baseuri = util.serverUri(server);
+
+  try {
+    const res = await nf(`${baseuri}/hello`);
```
Comment (justinbach) [inline test/acceptance/koa-basic-test.js]: What's the reason for this call? It doesn't seem to be playing any role in this test as it stands, and it appears to be 404ing.

**Review by justinbach — DISMISSED** [review]
LGTM, just noticed a few minor things along the way

File: examples/serverKoa.js (hunk 1 above)
Comment (inadarei) [reply examples/serverKoa.js]: yeap. fixing

File: test/acceptance/koa-basic-test.js (hunk 2 above)
Comment (inadarei) [reply test/acceptance/koa-basic-test.js]: fixed

File: test/acceptance/koa-basic-test.js (hunk 3 above)
Comment (inadarei) [reply test/acceptance/koa-basic-test.js]: None at all. It's a leftover copy/paste. Removing. Thanks

**Pull request merged**